src/controllers/auth.controller.ts

```diff
@@ -0,0 +1,58 @@
+import { Request, Response } from "express";
+import { PrismaClient } from "@prisma/client";
+import bcryptjs from "bcryptjs";
+import jwt from "jsonwebtoken";
+import { validationResult } from "express-validator";
+
+const prisma = new PrismaClient();
+
+export const createUser = async (req: Request, res: Response) => {
+  try {
+    const errors = validationResult(req);
```

Comment:
use better validation frameworks. 
If you can use data transfer objects.
For now you can use libraries like `zod`